docs/bigtable-client.rst

```diff
@@ -1,6 +1,12 @@
 Client
 ~~~~~~
 
+.. warning::
+
+    gRPC is required for using the Cloud Bigtable API. As of May 2016,
```

Comment:
Yea, I was just saying maybe it's better to do an include of sorts so we're not copy/pasting this 20 times?